pkg/kubelet/apis/config/types.go

```diff
@@ -281,6 +281,7 @@ type KubeletConfiguration struct {
 	ResolverConfig string
 	// RunOnce causes the Kubelet to check the API server once for pods,
 	// run those in addition to the pods specified by static pod files, and exit.
+	// Deprecated: no longer has any effect.
```

Comment:
"Deprecated" seems to suggest that it still works. 
BTW, If it's still in the API, we can error out if it's set.

Reply:
Added validation in `ValidateKubeletConfiguration` to check if the runonce mode is set. If it is set, the kubelet will stop running and output an error log.